server/request_sendrawtx.go

```diff
@@ -51,6 +56,14 @@ func (r *RpcRequest) handle_sendRawTransaction() {
 	r.logger.Info("[sendRawTransaction] sending raw transaction", "tx", r.tx.Hash(), "fromAddress", r.txFrom, "toAddress", AddressPtrToStr(r.tx.To()), "txNonce", r.tx.Nonce(), "txGasPrice", BigIntPtrToStr(r.tx.GasPrice()))
 	txFromLower := strings.ToLower(r.txFrom)
 
+	r.reqRecord.ethSendRawTxEntry.TxFrom = r.txFrom
+	r.reqRecord.ethSendRawTxEntry.TxTo = r.tx.To().String()
+	r.reqRecord.ethSendRawTxEntry.TxNonce = int(r.tx.Nonce())
+	r.reqRecord.ethSendRawTxEntry.TxData = r.tx.Data()
+	if len(r.tx.Data()) > signatureMethodByteLen {
+		r.reqRecord.ethSendRawTxEntry.TxSmartContractMethod = r.tx.Data()[:10]
```

Comment:
use the const instead of 10 manually (`[:10]`)

Reply:
updated

Reply:
`tx.Data` is a byte slice, the method name should be the first 4 bytes, not 10?